.dev/patches/drupal-skip-dev-env-from-extension-discovery-in-testing.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/core/tests/bootstrap.php b/core/tests/bootstrap.php
```

Comment:
Why do we have this patch here, and not referenced from d.o?

Reply:
Because it fixes an issue related to this setup as you can see from the comment in this patch.

Reply:
But isn't this issue a bug in Drupal?

Reply:
Well, yes and no :) TL;DR; No.

Drupal does not support this kind of folder structure that I used here out of the box therefore it is not a core bug. This is sightly related to the problem that drupal/core can not be run from vendor dir.

Reply:
The we are effectively forking the core, which is not a good thing. Can you upload this patch to d.o? Maybe as a feature request?

Reply:
Feature request to allow modules to contain a forked version of Drupal core for testing/development? Well, they will push back because it is an edge case... 

![image](https://user-images.githubusercontent.com/1755573/60429280-0ab15080-9bfb-11e9-9f0e-f041ec51ca9d.png)

and this patch is only used for testing/development so I do not see why having this as a local patch is a problem.

Reply:
Also, there is nothing generic in this solution and I do not see how this could be generalized. "If you can place Drupal core within a module/theme which folder should it be? core? .dev? build?"